charts/datadog/templates/_orchestrator_explorer_config.yaml

```diff
@@ -0,0 +1,42 @@
+{{- define "orchestratorExplorer-add-default-config" -}}
+
+{{- $useDefaultConfig := true -}}
+
+{{/*
+If custom config is provided, then we don't use default config.
+*/}}
+{{- range $k, $v := .Values.clusterAgent.confd -}}
+{{- if regexMatch "^orchestrator.yaml.*"  $k  -}}
```

Comment:
should this only apply with valid confd files, such as orchestrator.yaml.default and orchestrator.yaml ?

Reply:
I also updated this part to check `clusterAgent.advancedConfd` as well
https://github.com/DataDog/helm-charts/blob/64823a361eff2d4c4f5536b2d887ec081df42ee4/charts/datadog/templates/_orchestrator_explorer_config.yaml#L14-L21